src/linux/users.rs

```diff
@@ -20,7 +20,11 @@ pub fn get_users_list() -> Vec<User> {
         .filter_map(|line| {
             let mut parts = line.split(':');
             if let Some(username) = parts.next() {
-                let mut parts = parts.skip(2);
+                let mut parts = parts.skip(1);
+                let uid = parts
+                    .next()
+                    .and_then(|x| u32::from_str_radix(x, 10).ok())
+                    .unwrap_or_default();
```

Comment:
I've been thinking about this and I'm definitely not sure if this is the right way to handle it... If we can't get the `user_id`, maybe it'd be better to just skip this user? What do you think?

Reply:
I agree.  Moved to `if let` to skip if the parse fails.